src/components/Blockquote/Blockquote.js

```diff
@@ -149,7 +149,7 @@ Blockquote.propTypes = {
   /**
    * Show content formated as code
    */
-  code: PropTypes.bool,
+  // code: PropTypes.bool,
```

Comment:
This is the right prop. The `code` option from the `kind`prop needs to be removed.

Reply:
Sure